cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -543,7 +543,10 @@ var (
 				return err
 			}
 			paths := util.UpdateFieldMask(cmd.Flags(), setEndDeviceFlags, attributesFlags(), endDevicePictureFlags)
-			if len(paths) == 0 {
+			rawUnsetPaths, _ := cmd.Flags().GetStringSlice("unset")
+			unsetPaths := util.NormalizePaths(rawUnsetPaths)
+
+			if len(paths)+len(unsetPaths) == 0 {
```

Comment:
Let's check that there's no overlap between the two

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/2301/files#diff-06f9dcb39ac587641facf395bb27e8fbR554-R557